hail/python/hail/docs/install/macosx.rst

```diff
@@ -9,8 +9,7 @@ Install Hail on Mac OS X
 
   .. code-block::
 
-    brew tap homebrew/cask-versions
-    brew install --cask temurin8
+    brew install zulu@11
 
   You *must* pick a Java installation with a compatible architecture. If you have an Apple M1 or M2
```

Comment:
Since you're here, M3+ is now a thing so this could be reworded to something mroe future-proof like
```suggestion
  You *must* pick a Java installation with a compatible architecture.
  If, for example, your processor uses M-series Apple silicon
```